src/page-templates/content-type/content-type.tsx

```diff
@@ -1,421 +1,138 @@
     GridLayout,
     TypographyScale,
-    TextInput,
-    ESystemIconNames,
-    Button,
-    Select,
 } from '@mdb/flora';
-import { Grid } from 'theme-ui';
+import { h5Styles, pageWrapper } from '../../styled/layout';
+import { SearchBox, SearchResults, SortBox } from '../../components/search';
+import { desktopFiltersStyles } from './styles';
+import { FilterItem, FilterTagSection } from '@mdb/devcenter-components';
+import { FeaturedCardSection } from '../../components/card-section';
 
-import Results from '../../components/search/results';
+import { DesktopFilters, MobileFilters } from '../../components/search-filters';
+
+import LanguagesSection from './languages-section';
+import TechnologiesSection from './technologies-section';
+import ProductsSection from './products-section';
 import Hero from '../../components/hero';
 import RequestContentModal, {
     requestContentModalStages,
 } from '../../components/request-content-modal';
+import { useSearchMeta } from '../../hooks/search/meta';
+import { shouldRenderRequestButton } from './utils';
 import { CTAContainerStyles } from '../../components/hero/styles';
-
-import { DesktopFilters, MobileFilters } from '../../components/search-filters';
-
+import { NextSeo } from 'next-seo';
 import { ContentTypePageProps } from './types';
-import { desktopFiltersStyles, resultsStringAndTagsStyles } from './styles';
-import { h5Styles, pageWrapper } from '../../styled/layout';
-
-import {
-    searchBoxSortBarWrapperStyles,
-    searchBoxStyles,
-    sortBoxStyles,
-} from '../../components/search/styles';
-
-import { FeaturedCardSection } from '../../components/card-section';
-
-import LanguagesSection from './languages-section';
-import TechnologiesSection from './technologies-section';
-import ProductsSection from './products-section';
-
-import { getURLPath } from '../../utils/format-url-path';
 import useSearch from '../../hooks/search';
-import { hasEmptyFilterAndQuery, isEmptyArray } from '../../hooks/search/utils';
-import { FilterTagSection, FilterItem } from '@mdb/devcenter-components';
-import {
-    createInitialSearchData,
-    getResultData,
-    getResultIsValidating,
-} from '../../hooks/search/utils';
-
-import { shouldRenderRequestButton } from './utils';
-import { SearchItem } from '../../components/search/types';
-import { getCanonicalUrlWithParams, getMetaDescr } from '../../utils/seo';
-import {
-    sortByOptions,
-    DEFAULT_PAGE_SIZE,
-} from '../../components/search/utils';
+import { searchWrapperStyles } from '../../components/search/styles';
+import { isEmptyArray } from '../../hooks/search/utils';
 
 let pluralize = require('pluralize');
 
-const ContentTypePage: NextPage<ContentTypePageProps> = ({
+const heroCrumbs = [
+    {
+        text: 'MongoDB Developer Center',
+        url: '/',
+    },
+];
+
+// Debug for DEVHUB-1501 which is not yet replicable.
+// If data is empty, capture an exception for Sentry.
+const useEmptyDataDebug = (results: any, error: any) => {
```

Comment:
There was more to this debug code previously but since we're supplying a fallback to `useSWR` now it should be guaranteed that the data is non-empty.... but if for some reason it still is we still capture a Sentry report.